data_and_models/annotations/ner/analyze.py

```diff
@@ -75,31 +75,34 @@
 
 
 def report(input_path: Path, verbose: bool) -> List[Union[int, float]]:
+    """Create report of the annotations."""
     print(f"Analyzing {input_path}...")
 
-    results = []
+    results: List[float] = []
```

Comment:
```suggestion
    results: List[Union[int, float]] = []
```

`results` is what is returned by the function. The return type of the function should then be used.